funnel/templates/proposal.html.jinja2

```diff
@@ -211,9 +206,25 @@
                   {% endfor %}
                 {% endif %}
               {% endfor %}
-            </div>
 
-            <p>{{ renderform(statusform, 'proposal-status', action=proposal.url_for('status'), submit='Save') }}</p>
+              <div class="col-md-6 col-lg-6 col-sm-12 col-xs-12 pull-right">
+                <h3>Status:
+                    {{ proposal.state.label.title }}
+                    {% if proposal.session and proposal.state.CONFIRMED %} &amp; <mark><a class="" href="{{ proposal.session.url_for() }}">Scheduled</a></mark>{% endif %}
+                </h3>
+                <ul class="list-inline">
+                {% for name, transition in proposal.state.transitions.iteritems() %}
+                  <li>
+                    <form action="{{ proposal.url_for('transition') }}" method="post" class="form-inline">
```

Comment:
We aren't using `render_form` because of styling constraints?

Reply:
@shreyas-satish renderform renders SelectField as dropdown. We need buttons on the front end as they are actions.